packages/core/phase1/triggers/generate.ts

```diff
@@ -30,11 +43,16 @@ const generateSetOfTriggers = (
   return existingTriggers.concat(finalTriggers)
 }
 
-export default (annotatedHearingOutcome: AnnotatedHearingOutcome): Trigger[] => {
+export default (annotatedHearingOutcome: AnnotatedHearingOutcome, generatePostUpdateTriggers = false): Trigger[] => {
```

Comment:
`generatePostUpdateTriggers` is temporary until we move the pos-update triggers code to separate trigger files. This is to ensure that we don't break Phase 1.

Reply:
Why don't we just do this now so we avoid too many changes to phase 1?